### ISSUE TITLE
Rev- Fixes

### ISSUE THREAD

**Issue by cnlohr** [AEFeinstein/Super-2024-Swadge-HW#11]
- [x] Microphone is laaaaaaame.  Switch to newly available microphone - https://www.lcsc.com/product-detail/Microphones_INGHAi-GMI4013S-2C58DB_C529870.html
- [x] Swap buttons (left/down)
- [x] Add copper behind the touch pad
- [x] Remove unused parts (gyro/accel/magnet), pending conversation
- [x] Add dot indicating pin 1 on SAO
- [x] Add silkscreen art
- [x] Move rear board components at least 2mm from board edge
- [x] Part movement and reroute. Move microphone from the nose to the gap between the menu/pause buttons and acrylic. Move the third screw hole from around the display to the nose. Delete the fourth screw hole around the display. Reroute as necessary. See quick example:
![image](https://github.com/AEFeinstein/Super-2024-Swadge-HW/assets/231180/45e544ac-73d0-4750-aaf5-ce59c72e66d0)

**Comment by AEFeinstein** [comment]
And remove unused parts (magnetometer?)

**Comment by cnlohr** [comment]
Add dot indicating pin 1 on SAO

**Comment by AEFeinstein** [comment]
I strongly recommend adding copper behind the touch pad. I've noticed that the _back_ is touch sensitive on this rev, which isn't great.

**Comment by AEFeinstein** [comment]
And I think we can get some art for rev2

**Comment by cnlohr** [comment]
- Fill in copper under touch pad.

**Comment by AEFeinstein** [comment]
* cad972224555df497809eae80f3a85bdda3ab0e8 - Added dot to SAO
* a722288b943a128ea8e3cc8c8a3047e676b30b1c - Removed keepout behind touchpad
* c49fef01a56f8b0b280d5c99c38fbb450975cad4 - Swapped left and down buttons

**Comment by eriktronic** [comment]
<img width="318" alt="Screenshot 2023-08-26 210917" src="https://github.com/AEFeinstein/Super-2024-Swadge-HW/assets/48095517/d5e33a78-224a-4317-b76a-2f6e39a238e7">
Move rear board components at least 2mm from board edge

**Comment by cnlohr** [comment]
Change Log Summary, So far, in my new PR:
1. Move Microphone
2. Switch microphone for new mic.
3. Switch support circuitry for new mic.
4. Deleted extra screw.
5. Moved components on back in so that none are within 1mm of the edge.
6. Tweaked power for gyro.
7. Removed rumble drive circuitry.
8. Tweaked footprint on magnetometer.

Additional: Shrunk touch pad, it is now 24.5mm across.
Additional: Not sure why pinout on touchpad appeared different on updated touch pad - this should make them the same.


TODO:
- [ ] Add Silkscreen Art
- [ ] Pending, remove unused circuitry

These changes are in https://github.com/AEFeinstein/Super-2024-Swadge-HW/pull/14

**Comment by AEFeinstein** [comment]
As noted in #14, once you remove unused circuitry give me the mutex and I'll apply art

**Comment by cnlohr** [comment]
@AEFeinstein  Should I try using a smaller inductor for the display? It feels weird being the biggest chonker on the swadge.

**Comment by cnlohr** [comment]
#14 Mutex released.

**Comment by cnlohr** [comment]
Also - so you are aware, my new strategy is set min spacing to 7.2mil for DRC, and 8.2 for design.  

That magnetometer is annoyying.

**Comment by cnlohr** [comment]
Ugh... inductor wise, let's keep it.  All the other suitable ones look like they could snag. https://www.lcsc.com/product-detail/Inductors-SMD_Murata-Electronics-LQH32PZ100MNCL_C464437.html

**Comment by AEFeinstein** [comment]
Keeping the more cube-ular inductor is A-OK.

**Comment by AEFeinstein** [comment]
@cnlohr I'll make my small adjustments and art and let you know when it's good to order. My edits are done on the google docs RFQ, and I think you have some pending.

**Comment by AEFeinstein** [comment]
@cnlohr, I added art, tweaked some positions, and fixed DRC errors and warnings. I am done for the evening.

> Also - so you are aware, my new strategy is set min spacing to 7.2mil for DRC, and 8.2 for design.

FYI 8.2mil DRC for design threw a ton of errors, so I bumped it down to 0.2mm (7.87mil), which it looks like it was actually designed with. I also reassigned all track widths to their netclass width. It's nice.

> That magnetometer is annoyying.

Yes! I synced the board with `Swadge-Parts` so the footprint in use has pill-shaped pads, not rounded-square ones. I also overrode clearances for those four pads. I don't know what you want to do, but that'll be a quick chat before ordering.
